.github/workflows/build-assets.yml

```diff
@@ -87,6 +87,10 @@ jobs:
     strategy:
       matrix:
         include:
+          - os: ubuntu-24.04
```

Comment:
Should this have a name?  Same for 22.04?

Reply:
No. The 'name' is a way for us to label the builds so we can tell at a glance what special flags are turned on in the github actions' job tab.  Anything that doesn't have a special name like LN, k8, or a combination of the two is a standard platform build.